CHANGELOG.md

```diff
@@ -640,3 +642,5 @@ await myHelper(context);
 [0.12.0]: https://github.com/coda/packs-sdk/compare/v0.11.0...v0.12.0
 [0.11.0]: https://github.com/coda/packs-sdk/compare/v0.9.0...v0.11.0
 [0.9.0]: https://github.com/coda/packs-sdk/compare/v0.8.2...v0.9.0
+
+[1.6.0]: https://github.com/coda/packs-sdk/compare/v1.4.1...v1.6.0
```

Comment:
I thought I had verified this behavior was fixed at some point, but the link is getting added at the end instead of the top again

Reply:
The base of this comparison should also be 1.5.1, not 1.4.1

Reply:
Fixed by force for now